VContainer/Assets/VContainer/Tests/Fixtures.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using Unity.Entities;
```

Comment:
Maybe  we need the `#if ` directive ?

Reply:
This is my mistake. i had remove this line.